src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  box-sizing: border-box;
```

Comment:
lets pass `box-sizing: border-box;` to `*` selector
and setting `line-height` for the whole html selector is not good practice